### DISCUSSION TITLE
Repeating words distribution in train set

### DISCUSSION THREAD

**Discussion by pooja-kabra** [PaddlePaddle/PaddleOCR#12071] [Q&A]
I am finetuning the text recognition network on product labels. Some text fields occur repeatedly on the labels. For example, PRODUCT_ID: 343765. All labels will have the text 'PRODUCT_ID'. How should the word distribution be in the training set for these texts? For example in a 5k dataset, how many samples should these repeating words take up?
